media-sound/ocp/ocp-0.1.22_git.recipe

```diff
@@ -1,70 +1,83 @@
 SUMMARY="Open Cubic Player"
-DESCRIPTION="
-ocp - Open Cubic Player, a music player ported from DOS.
-"
+DESCRIPTION="ocp - Open Cubic Player, a music player ported from DOS."
 HOMEPAGE="http://stian.cubic.org/project-ocp.php"
-SOURCE_URI="git://git.code.sf.net/p/opencubicplayer/code#f97d87f960624de013826e9c2970404f90eef211"
-REVISION="1"
+COPYRIGHT="1994-2016 Niklas Beisert, Stian Skjelstad and others"
 LICENSE="GNU GPL v2"
-COPYRIGHT="1994-2012 Niklas Beisert, Stian Skjelstad and others"
-##		media-libs/flac >= 1.2.1"
-## ncurses zlib ... cf. ocp.spec
+REVISION="1"
+gitRevision="835bf803f590512848282bbf74986ebfaec17da3"
+SOURCE_URI="https://sourceforge.net/code-snapshots/git/o/op/opencubicplayer/code.git/opencubicplayer-code-$gitRevision.zip"
+CHECKSUM_SHA256="8691b9fa9f512c98307c6e4fb2723c10a4a8f212da8a3af1b9d15d2fe32764fb"
+SOURCE_DIR="opencubicplayer-code-$gitRevision"
 
-ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
 PROVIDES="
 	ocp$secondaryArchSuffix = $portVersion
-	cmd:ocp = $portVersion
-	cmd:ocp_0.1.21 = $portVersion
+	cmd:ocp$commandSuffix = $portVersion
+	cmd:ocp_0.1.22$commandSuffix = $portVersion
```

Comment:
If you can rename the recipe to `ocp-0.1.22~git.recipe` and:
```diff
-	cmd:ocp_0.1.22$commandSuffix = $portVersion
+	cmd:ocp_${portVersion#~*}$commandSuffix
```
(I did not test this.)